src/ui/zcl_abapgit_gui_chunk_lib.clas.abap

```diff
@@ -46,75 +46,74 @@ CLASS zcl_abapgit_gui_chunk_lib DEFINITION
       IMPORTING
         !io_news       TYPE REF TO zcl_abapgit_news
       RETURNING
-        VALUE(ro_html) TYPE REF TO zcl_abapgit_html
+        VALUE(ri_html) TYPE REF TO zif_abapgit_html
```

Comment:
An instance of interface is not object? Why did you change `ro` to `ri`?

Reply:
prefixes naming convention, https://docs.abapgit.org/development/guidelines.html, its a reference to interface

Reply:
I do understand you have a guidelines, but what is the difference between a reference to class and a reference to interface? ABAP is kinda "every second something unexpectedly new".